### DISCUSSION TITLE
Steno Missing

### DISCUSSION THREAD

**Discussion by sinungp** [Security-Onion-Solutions/securityonion#6551] [Unsupported Versions] [answered]
Hello

I want to ask, why is the stenographer error in security onion 2.3.90, I use securityonion on the local computer

After I checked the log, this report came out

**2021-12-14T07:51:32.103560Z T:0f8867 [stenotype.cc:567] Starting, page size is 4096
2021-12-14T07:51:32.103625Z T:0f8867 [stenotype.cc:594] Setting up AF_PACKET sockets for packet reading
2021-12-14T07:51:32.766266Z T:0f8867 [stenotype.cc:619] CHECK(SUCCEEDED(__check_success_error__)) builder.Bind(flag_iface, &v3): No such device
ABORTABORTABORT
/usr/bin/stenotype() [0x4081f8]
/usr/bin/stenotype() [0x42e4a4]
/lib64/libc.so.6(__libc_start_main+0xf5) [0x7fa50dde5555]
/usr/bin/stenotype() [0x40412b]
2021/12/14 07:51:32 Stenotype stopped after 742.320323ms: stenotype wait failed: signal: aborted
2021/12/14 07:51:32 Stenotype ran for too little time, crashing to avoid stenotype crash loop**

Is there anything I can do to fix this?

thanks

**Comment by InfosecGoon** [comment]
Are any of the other monitoring services (like Zeek or Suricata) working properly? That "No such device" error makes me think that you've got the monitoring interface set to something that doesn't exist.

**Reply by sinungp** [reply]
i already confirm that the interface (bond0) is running, i have tested in tcpdump also

**Comment by dougburks** [comment]
@sinungp Please provide more information:
- What are the hardware specs of your machine?
- How many NICs do you have?
- What kind of NICs are they?
- What options did you choose during Setup?

**Reply by sinungp** [reply]
Hello 

-My machine specs : 
1. core i3
2. 12gb ddr4
3. ssd SATA 240gb

-I use 2 NIC

-The type of NIC
1. RTL8111/8168/8411 PCI Express Gigabit Ethernet Controller
2. D-Link System Inc DGE-528T Gigabit Ethernet

- I use standalone mode 

Thank you

**Reply by dougburks** [reply]
Not sure why you'd be seeing that. You could try different NICs or a different machine altogether.

**Comment by sinungp** [comment, marked as answer]
I have solved this problem by reinstalling security onion

thank you
